optimize_versions.json

```diff
@@ -1 +1 @@
-["3.10.0", "3.9.0", "3.8.0", "3.7.0"]
```

Comment:
only includes the specific archived version! Associated with this change would be removing all the source files for those other versions.